packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/scim/test-instance.md

```diff
@@ -0,0 +1,73 @@
+
+After you've enabled the provisioning API connection in your test instance, configure user attribute mappings to and from Okta in the **Provisioning** tab of your instance:
+
+* **To App**: User attribute mappings from Okta to your app
+* **To Okta**: User attribute mappings from your app to Okta
+
+1. Select **To App** on the left **Settings** panel of the **Provisioning** tab.
+  The **Provisioning to App** settings appear. The provisioning operations are already set by default from the [SCIM properties](#properties) section when you configured your integration.
+
+1. Scroll to the **{yourApp} Attribute Mappings** section.
+
+   * Delete attributes:
+     1. Click **X** next to the attribute that you want to delete, and then click **OK** to confirm.
+
+        Repeat this step until you remove all the mappings for the attributes that you want to delete.
+
+     1. After removing all the mappings for the attributes that you want to delete, click **Go to Profile Editor**.
+
+     1. In the Profile Editor, delete all the corresponding attributes from the mapping: click **X** next to the attribute, and then click **Delete Attribute** to confirm.
+
+        Repeat this step for all the attributes that you want to delete.
+
+   * Add attributes:
+
+     1. In the Profile Editor, click **Add Attribute**.
+
+     1. Enter the information for the new attribute that you’re adding and then click **Save**.
+
+        > **Note:** The **Scope** property determines whether the attribute that you're adding can be assigned at a group level or per user. If you want your admins to assign a value for this attribute at a group level, don't select the **User personal** checkbox.
+
+     1. After adding attributes, go back to the **{yourApp} Attribute Mappings** section and click **Edit** to map your new attributes.
+
+     1. In the dialog that appears, there are two dropdown fields. In the first dropdown list, select **Map from Okta Profile**. In the second dropdown list, choose the Okta profile attribute that you want to map over to the SCIM attribute. Click **Save**.
+
+          Repeat this step for all SCIM attributes that you want to map (from Okta to your app).
+
+     <div class="three-quarter border">
+
+     ![Displays the map attribute dialog.](/img/oin/scim_check-attributes-14.png)
+
+     </div>
+
+     5. After you update the mappings from Okta to your app, click **To Okta** in the **Settings** section.
+
+     6. Scroll to the **{yourApp} Attribute Mappings** section. Look for the attribute that you want to update and click **Edit**.
+
+     7. In the dialog that appears, there are two dropdown fields. In the first dropdown list, select **Map from {yourApp} App Profile**. In the second dropdown list, choose the SCIM attribute that you want to map to the Okta attribute. Click **Save**.
```

Comment:
What's the name of the dialog and the two fields? We could structure the steps like this:

1.  Scroll to the **{yourApp} Attribute Mappings** section. Fine the attribute that you want to update and click **Edit**. The `<dialog name>` opens.
2. Select **Map from {yourApp} App Profile** from the `<field name>` dropdown list. 
3. From the `<field name>` dropdown list, select the SCIM attribute that you want to map to the Okta attribute. 
4. Click **Save**.

Reply:
@barbaravo-okta : I've updated this section based on your suggestion. Please review again.